CorsixTH/Lua/rooms/training.lua

```diff
@@ -248,4 +270,13 @@ function TrainingRoom:onHumanoidLeave(humanoid)
   Room.onHumanoidLeave(self, humanoid)
 end
 
+function TrainingRoom:afterLoad(old, new)
+  if old < 178 then
```

Comment:
Causes crash if room is being edited on save. Shouldn't impact testing.  (Do we have a convention that the `if old ...` line should only contain that check?)
```suggestion
  if old < 178 and self.built then
```